ai-ml/trainium-inferentia/main.tf

```diff
@@ -39,7 +47,7 @@ data "aws_ecrpublic_authorization_token" "token" {
 }
 
 locals {
-  name   = var.name
+  name   = "${var.name}-${random_string.this.result}"
```

Comment:
remove this and put the name back to `var.name` and keep the name as `trainium-inferentia`.

Reply:
done